mods/persistence/modules/chargen/storage/science_boxes.dm

```diff
@@ -27,7 +27,7 @@
 		/obj/item/scanner/health = 1
 	)
 
-/obj/item/chargen_box/science/burn_toxin
+/obj/item/chargen_box/science/burn_toxin // now deprecated + unused, see replacement medpacks below
```

Comment:
This can be removed if it's no longer used anywhere.

Reply:
Okie dokie